### ISSUE TITLE
Publish to Windows Store

### ISSUE THREAD

**Issue by daviset1** [tomlandia/fuckr#111]
Hi guys,

Asking if we can use the Desktop App Converter tool to wrap this as an .appx and publish to the Windows 10 Store for easy install/uninstall. There used to be a nice Windows 10 UWP made by @shyrat in the Store, but sadly hasn't been updated since the API change. Now that Fuckr is working again, would be nice to put in the store now that Win32s are allowed.

Thanks

**Comment by seasider620** [comment]
Regarding to the qeuestion, is it legal to develop and publish applications that use Grindr Services? I would like to create an application like Fuckr, but for Android, so I would like to know if it is legal to develop and publish it on Google Play.

**Comment by mentaluproar** [comment]
I used to use a client called guyspy that I got from the windows app store.

**Comment by daviset1** [comment]
Yes, it's legal. There are already 2 third party Grindr apps on Windows:

- **Meet'm** (developed by OABSoftware), which works great today but it's a Silverlight 8.1 app and only available on mobile handset

- **Shyguy** (developed by @shyrat) is a universal Windows 10 app and USED to work great (beautiful adaptive fluid UI, synced/roamed your conversations across your devices), but it is currently not working because Cody hasn't updated it with the API fix yet.

Thus the only desktop Grindr solution that's currently working is fuckr, which is why I'm asking for it to be packaged and brought to the Windows Store with Project Centennial.
